src/CattyTests/Parser & Serialization/CatrobatLanguage0.95/XMLParserTests/Unit Tests/XMLParserBrickTests095.swift

```diff
@@ -34,18 +34,6 @@ final class XMLParserBrickTests095: XMLAbstractTest {
         parserContext = CBXMLParserContext(languageVersion: CGFloat(Float32(0.95)), andRootElement: GDataXMLElement())
     }
 
-    func testInvalidSetVariableBrickWithoutFormula() {
```

Comment:
Could you please give me a short info why these tests should be deleted?

Reply:
Looks like we are running into a regression, so removing these tests is probably not a good idea. I have made a few suggestions.